lib/galaxy/datatypes/test/affy_v_3.CEL

```diff
@@ -0,0 +1,74 @@
+[CEL]
```

Comment:
This file needs to end with `.cel ` for the test to work. Make sure to also adjust the doctests for the new filename.

Reply:
Done. Awesome thank you

Reply:
Thanks @mvdbeek. Hope it will make it to 19.05.